docs/conf.py

```diff
@@ -32,11 +32,14 @@
 
 import os
 import sys
+from typing import Dict
 
+import sphinx_rtd_theme
```

Comment:
Are these necessary? Sorry, I cannot find lines using them. 🤔

Reply:
let me check. 
Regarding the documents, it would be better to have another PR

Reply:
```
# The theme to use for HTML and HTML Help pages.  See the documentation for
# a list of builtin themes.
#
#html_theme = 'alabaster'
html_theme = 'sphinx_rtd_theme'
```
I supposed the theme is used in this but it would be dynamically loaded. I'll test it and let you know

Reply:
I checked that this is not required. thank you for the review! Fixed.